features/step_definitions/networking.rb

```diff
@@ -1157,17 +1157,21 @@
   end
 end
 
-Given /^I check load-sctp-module in all nodes$/ do 
+Given /^I check load-sctp-module in #{NUMBER} workers$/ do | workers_num |
   ensure_admin_tagged
   _admin = admin
-  env.nodes.each do |node|
-    @result = node.host.exec_admin("lsmod \| grep sctp")
+  $i = 0
+  $num = Integer(workers_num)
+  while $i < $num  do
+    @result = cb.workers[$i].host.exec_admin("lsmod \| grep sctp")
```

Comment:
we can check modules directly in `/sys/modules`

`/sys/module/sctp/initstate`  shows `live`

```bash
sh-4.4# cat /sys/module/sctp/initstate
live
```

this way we don't have to dump all the modules and grep, because the grep can match substrings as well unless we anchor it, but `lsmod` is just messy in general.  It matches the `libcrc32c` line for example.

```bash
      'lsmod\ \|\ grep\ sctp
      sctp                  401408  32
      libcrc32c              16384  5 nf_conntrack,nf_nat,openvswitch,xfs,sctp
```

`/sys/module/sctp/initstate` is cleaner

```ruby
@result = cb.workers[$i].host.exec_admin("cat /sys/module/sctp/initstate")
unless @result[:response].include? "live"
```

Reply:
Ruby is fancy, but I think the loop could be simplified, using `.to_i` and `.times`

```ruby
  workers_num.to_i.times do |i|
    @result = cb.workers[i].host.exec_admin("cat /sys/module/sctp/initstate")
    unless @result[:stdout].to_s.include? "live"
      raise "no sctp module"
    end
  end
```

Reply:
Fixed by using "cat /sys/module/sctp/initstate", thanks!